content/en/xlt/load-testing/Manual/320-test-evaluation.md

```diff
@@ -159,17 +159,19 @@ The Page Load Timings section offers deeper insight into the page loading perfor
 These timings will be recorded only when using `XltChromeDriver` or `XltFirefoxDriver` to run the browser. These are special WebDriver implementations that install an extension into the browser which is able to gather all the timings and report them to XLT.
 {{% /note %}}
 
-### Custom Timers & Values
+### Custom Timers and Custom Data
```

Comment:
How about splitting this section into two sections, "Custom Timers" and "Custom Data"? This way, each report page has a corresponding section in the manual.